torchaudio/compliance/kaldi.py

```diff
@@ -904,80 +818,39 @@ def resample_waveform(waveform: Tensor,
 
     Args:
         waveform (Tensor): The input signal of size (c, n)
-        orig_freq (float): The original frequency of the signal
-        new_freq (float): The desired frequency
+        orig_freq (int): The original frequency of the signal
+        new_freq (int): The desired frequency
```

Comment:
If we are changing the docstring, I also suggest to change the annotation to `int`. This is kind of BC-breaking but it is an enforcement as it was expected in the original implementation too.

Reply:
Yes, I forgot to do it but intended too.